Classes/URKArchive.mm

```diff
@@ -163,7 +163,7 @@ - (instancetype)initWithFile:(NSURL *)fileURL password:(NSString*)password error
         _threadLock = [[NSObject alloc] init];
 
         if (bookmarkError) {
-            URKLog("Error creating bookmark to RAR archive: %{public}@", bookmarkError);
+            URKLogError("Error creating bookmark to RAR archive: %{public}@", bookmarkError);
```

Comment:
Good catch! I copied this from `UZKArchive`, where a file not existing at init time is expected, since archives are writeable. I would say though that it should be **Fault**-level, not **Error**:

https://github.com/abbeycode/UnrarKit/tree/v2.9#fault-log-level